packages/venia-concept/src/components/FilterModal/FilterFooter/filterFooter.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import catalogActions, { serialize } from 'src/actions/catalog';
+import { withRouter } from 'react-router-dom';
+import classify from 'src/classify';
+import defaultClasses from './filterFooter.css';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+class FilterFooter extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            resetButton: PropTypes.string,
+            resetButtonDisabled: PropTypes.string,
+            applyButton: PropTypes.string,
+            applyButtonDisabled: PropTypes.string,
+            footer: PropTypes.string
+        }),
+        history: PropTypes.object,
+        filterClear: PropTypes.func,
+        chosenFilterOptions: PropTypes.object,
+        closeModalHandler: PropTypes.func
+    };
+
+    static getDerivedStateFromProps(nextProps) {
+        const { chosenFilterOptions } = nextProps;
+        let filterOptionsArePristine = true;
+
+        for (const optionName in chosenFilterOptions) {
+            if (chosenFilterOptions[optionName].length > 0) {
+                filterOptionsArePristine = false;
+                break;
+            }
+        }
+
+        return { areOptionsPristine: filterOptionsArePristine };
+    }
+
+    state = {
+        areOptionsPristine: true
+    };
+
+    resetFilterOptions = () => {
+        const { history, filterClear } = this.props;
+        history.push();
+        filterClear();
+    };
+
+    handleApplyFilters = () => {
+        const { history, chosenFilterOptions, closeModalHandler } = this.props;
+        history.push('?' + serialize(chosenFilterOptions));
```

Comment:
Can we use the `filterAdd` function here?

Reply:
This only updated the URL and closes the modal, it doesn't affect the redux state. I'm thinking on adding "appliedFilters" redux state that will keep track of applied filters which will affect the category page listing. We'll use that function here.